docs/storefront/stencil/themes/context/object-reference/schemas/settings.yml

```diff
@@ -179,6 +183,7 @@ properties:
       user_agent:
         type: string
         description: User agent string of the request.
```

Comment:
An alternative is using JavaScript detection https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent